zeek/plugin/src/plugin.cc

```diff
@@ -303,6 +313,14 @@ void plugin::Zeek_Spicy::Plugin::InitPostScript() {
             ::zeek::EventHandlerPtr handler =
                 ::spicy::zeek::compat::event_register_Register("spicy_analyzer_for_mime_type");
             ::spicy::zeek::compat::event_mgr_Enqueue(handler, vals);
+        };
+
+        for ( auto mt : p.mime_types )
+            register_analyzer_for_mime_type(tag, mt);
+
+        if ( p.parser ) {
+            for ( auto mt : p.parser->mime_types )
+                register_analyzer_for_mime_type(tag, mt);
```

Comment:
Do we have anything which ensures that we register a mime type exactly once? If not it might make sense to e.g., here track in a set which mime types are already registered.

Reply:
it's actually ok to register the same MIME type multiple times, Zeek is fine with that and triggers them all.